### PR TITLE
use design token values within web-components

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

File: libs/web-components/src/variables.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+:root {
```
Comment (ArakTaiRoth) [inline libs/web-components/src/variables.css:1]: Just wondering, would it be better if we had these variables coming in from the design docs instead, through the design tokens?

File: package.json [hunk 2]
```diff
@@ -71,6 +71,7 @@
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
+    "@abgov/design-tokens": "^1.0.5-dev.4",
```
Comment (ArakTaiRoth) [inline package.json:74]: Just wondering the order of operations we want to take for this. I was thinking once Tom approves this work, then we merge design tokens to production, and change this line to use production?

Comment (chrisolsen) [reply package.json:74]: I don't think the design tokens are stable to the point were it should be released, so we will have to keep it referring to the dev branch until that happens.

**Review by ArakTaiRoth — APPROVED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**